google/google-how-to/gce/check-cve-on-instance.rst

```diff
@@ -0,0 +1,88 @@
+How to know if the Ubuntu image you have launched has addressed a specific CVE?
+===============================================================================
+
+| This "how-to" will help you determine if a running Ubuntu instance on GCP has addressed a specific CVE you are interested in.
+| We will heavily rely on the OVAL tooling developed by the Canonical Security Team to address this question.
+| This guide assumes you have SSH access to an Ubuntu instance running on GCP. That being said, this guide should work for any Ubuntu host where you have command line access.
+
+Install the required packages
+-----------------------------
+
+Run the following command on the instance to install the required packages.
+
+.. code::
+
+    sudo apt update && sudo apt install libopenscap8 bzip2 -y
+
+Scanning the instance
+---------------------
+
+Follow the :ref:`next section<Scanning an Ubuntu Pro instance>` if you are running an Ubuntu Pro instance.
+
+The following commands will:
+
+#. Download the CVE OVAL feed for the release your instance is running.
+#. Extract the XML file from the downloaded archive.
+#. Use the :code:`oscap` tool to run an OVAL eval using the downloaded OVAL feed.
+
+.. code::
+
+    wget "https://security-metadata.canonical.com/oval/com.ubuntu.$(lsb_release -cs).cve.oval.xml.bz2"
+    bunzip2 "com.ubuntu.$(lsb_release -cs).cve.oval.xml.bz2"
+    oscap oval eval --report report.html "com.ubuntu.$(lsb_release -cs).cve.oval.xml"
+
+Scanning an Ubuntu Pro instance
+-------------------------------
+
+An Ubuntu Pro instance is configured with ESM (Extended Security Maintenance) apt repositories in addition to the default main repository. On Pro instances :code:`apt` will try to download packages from an ESM repository. If :code:`apt` can't find the package there, it will fallback to the main repository.
+
+We need to download and merge the OVAL feeds of the repositories configured on the instance before running a scan.
+
+Install :code:`oval-xml-feed-merge` on the instance to merge OVAL feeds.
+
+.. code::
+
+    sudo snap install oval-xml-feed-merge
+
+Download the OVAL feeds. The merge tool can access only the home directory so please download the OVAL feeds there. The merge tool can take upto a few minutes if you are running an instance with a conservative CPU configuration.
+
+.. code::
+
+    wget "https://security-metadata.canonical.com/oval/com.ubuntu.esm-apps_$(lsb_release -cs).cve.oval.xml.bz2"
```

Comment:
nit: `/etc/os-release` is the more "modern" way of getting all the information rather than `lsb_release`. It's a slow phase out, so probably best to start referencing.

it's also a nit because `lsb_release` isn't _guaranteed_ to be installed, but `/etc/os-release` is guaranteed to be generated. an example there is a docker container.

Reply:
Now using `/etc/os-release` instead of `lsb_release`